conferences/2022/devops.json

```diff
@@ -972,5 +972,18 @@
     "cocUrl": "https://devopsdays.org/events/2022-buffalo/conduct",
     "offersSignLanguageOrCC": true,
     "url": "https://devopsdays.org/events/2022-buffalo/welcome"
+  },
+  {
+    "name": "JNation",
+    "startDate": "2022-06-07",
+    "endDate": "2022-06-07",
+    "city": "Coimbra",
+    "country": "Portugal",
+    "online": false,
+    "cfpUrl": "https://jnation.pt/cfp",
+    "cfpEndDate": "2022-05-01",
+    "twitter": "@jnation",
```

Comment:
should be

`"twitter": "@jnationConf",`

Reply:
Ok. I've done it here: https://github.com/tech-conferences/conference-data/pull/4369